pkg/account/server.go

```diff
@@ -50,27 +56,30 @@ type Component interface {
 
 type server struct {
 	c             Component
-	config        oauth.Config
+	config        Config
 	store         account_store.TransactionalInterface
 	session       sess.Session
 	generateCSP   func(config *oauth.Config, nonce string) string
 	schemaDecoder *schema.Decoder
 }
 
 // NewServer returns a new account app on top of the given store.
-func NewServer(c *component.Component, store account_store.TransactionalInterface, config oauth.Config, cspFunc func(config *oauth.Config, nonce string) string) (Server, error) {
+func NewServer(c *component.Component, store account_store.TransactionalInterface, oauthConfig oauth.Config, sessionConfig sess.Config, cspFunc func(config *oauth.Config, nonce string) string) (Server, error) {
```

Comment:
This func signature isn't getting much better if we keep adding arguments. Maybe it's time to start taking a config struct (you're already introducing a new `Config` struct) or options instead.